sql/6_0_0-to-6_1_0_upgrade.sql

```diff
@@ -585,12 +585,13 @@ INSERT INTO `ccda_sections` (`ccda_sections_id`, `ccda_components_id`, `ccda_sec
 #EndIf
 
 #IfNotRow2D list_options list_id lists option_id Care_Team_Status
-INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`) VALUES ('lists', 'Care_Team_Status', 'Care Team Status', '1');
-INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`) VALUES ('Care_Team_Status','active','Active',10,0,0);
-INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`) VALUES ('Care_Team_Status','inactive','Inactive',20,0,0);
-INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`) VALUES ('Care_Team_Status','suspended','Suspended',30,0,0);
-INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`) VALUES ('Care_Team_Status','proposed','Proposed',40,0,0);
-INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`) VALUES ('Care_Team_Status','entered-in-error','Entered In Error',50,0,0);
+INSERT INTO list_options (list_id,option_id,title, seq, is_default, option_value, notes) VALUES ('lists','Care_Team_Status','Care Team Status',0, 1, 0, 'This list originally comes from http://hl7.org/fhir/R4/valueset-care-team-status.html');
+INSERT INTO list_options (list_id,option_id,title,seq,is_default,activity,notes) VALUES ('Care_Team_Status','proposed','Proposed',10,0,1, 'The care team has been drafted and proposed, but not yet participating in the coordination and delivery of patient care.');
+INSERT INTO list_options (list_id,option_id,title,seq,is_default,activity,notes) VALUES ('Care_Team_Status','active','Active',20,1,1, 'The care team is currently participating in the coordination and delivery of care.');
+INSERT INTO list_options (list_id,option_id,title,seq,is_default,activity,notes) VALUES ('Care_Team_Status','suspended','Suspended',30,0,1, 'The care team is temporarily on hold or suspended and not participating in the coordination and delivery of care.');
+INSERT INTO list_options (list_id,option_id,title,seq,is_default,activity,notes) VALUES ('Care_Team_Status','inactive','Inactive',40,0,1, 'The care team was, but is no longer, participating in the coordination and delivery of care.');
+INSERT INTO list_options (list_id,option_id,title,seq,is_default,activity,notes) VALUES ('Care_Team_Status','entered-in-error','Entered in Error',50,0,1, 'The care team should have never existed.');
```

Comment:
@adunsulag ,
What is the timeframe of these changes (ie. how long ago were the original entries placed)? And are there any updates in this that are critical (ie. if a user is running of development codebase and does not get these changes into their database)?

Reply:
i ended up migrating these changes to the end of the sql upgrade script in the follow up PR